sitemap/sitemap_test.go

```diff
@@ -122,7 +122,7 @@ func expectedURLData() []URL {
 		sitemapUrls[i].XMLName.Space = "http://www.sitemaps.org/schemas/sitemap/0.9"
 		sitemapUrls[i].XMLName.Local = "url"
 		sitemapUrls[i].ChangeFreq = "monthly"
-		sitemapUrls[i].LastMod = "2022-03-01T00:00:00.000Z"
+		sitemapUrls[i].LastMod = "2022-04-01T00:00:00.000Z"
```

Comment:
Instead of changing dates manually, you can add some code to get month's first date